pkg/sysinfo/sysinfo_linux.go

```diff
@@ -211,6 +213,24 @@ func applyCPUSetCgroupInfo(info *SysInfo, cgMounts map[string]string) []string {
 	return warnings
 }
 
+// applyHugetlbCgroupInfo reads the cpuset information from the cpuset cgroup mount point.
+func applyHugetlbCgroupInfo(info *SysInfo, cgMounts map[string]string) []string {
```

Comment:
This doesn't appear to be called from anywhere?

Reply:
This is called from `func New(quiet bool) *SysInfo`